exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Schema.java

```diff
@@ -16,21 +16,22 @@
 
 package com.exonum.binding.core.service;
 
-import com.exonum.binding.common.hash.HashCode;
-import java.util.List;
+import com.exonum.binding.core.storage.indices.ProofListIndexProxy;
+import com.exonum.binding.core.storage.indices.ProofMapIndexProxy;
 
 /**
  * A schema of the collections (a.k.a. indices) of a service.
+ *
+ * <p>To verify the integrity of the database state on each node in the network,
```

Comment:
I don't know where is the proper place for this doc, but in Service it looks weird.